src/test/spec/crud.rs

```diff
@@ -18,6 +18,11 @@ fn test_predicate(test: &TestCase) -> bool {
     // The Rust driver doesn't support unacknowledged writes.
     let lower = test.description.to_lowercase();
 
-    // TODO: RUST-1071, RUST-1215: unskip comment tests
-    !lower.contains("unacknowledged") && !lower.contains("comment")
+    // TODO: RUST-1071: unskip comment tests
+    // RUST-1215: unskipped comment tests for estimatedDocumentCount
+<<<<<<< HEAD
```

Comment:
looks like there were some git issues here -- let me know if you need any help fixing these!